app/src/main/java/com/example/mylittlelibrary/data/Book.kt

```diff
@@ -1,5 +1,12 @@
 package com.example.mylittlelibrary.data
-
+import com.google.gson.annotations.SerializedName
 import java.util.*
 
-data class Book(var name : String, var lendTo: String, var date: Date)
+data class Book(@SerializedName("name") var name : String, var lendTo: String, var date: Date)
```

Comment:
Add @SerializedName for the project URL:https://howtodoinjava.com/gson/gson-serializedname/

Reply:
Ahhhhhhh... it's not adding value if it's the same name (name) and capitalization as the attribute it is describing (name).